gef-ui/server/controllers/application-details/index.js

```diff
@@ -202,6 +203,10 @@ const applicationDetails = async (req, res, next) => {
   try {
     const application = await Application.findById(dealId, user, userToken);
 
+    if (application.status === CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS) {
+      res.render('partials/review-decision.njk', { applicationStatus: application.status });
```

Comment:
- I don't think these 3 new lines are required
- Usually `res.render` used as return from controller. I am not sure calling it here works.
- I know that test is just for status `UKEF_APPROVED_WITHOUT_CONDITIONS`, but review decision is also required for other statuses.

Reply:
If the above lines are removed this test renders review-decision` when page requested is `review-decision fails. mockresponse.render expects a condition to check application status and then render the template.

Reply:
Does this work if you test it manually?
Did you try to create manual Deal that is approved without conditions?
Also please add console.log to understand if you are really going to this code branch.
P.S. Controller shouldn't be modified just to pass tests.

Reply:
Added console.log

Reply:
I removed these 3 lines